src/NexusMods.DataModel/Services.cs

```diff
@@ -44,39 +45,21 @@ public static IServiceCollection AddDataModel(this IServiceCollection coll)
         coll.AddSettingsStorageBackend<MnemonicDBSettingsBackend>(isDefault: true);
         coll.AddAttributeCollection(typeof(Setting));
 
-        coll.AddSingleton<MnemonicDB.Storage.InMemoryBackend.Backend>();
-        coll.AddSingleton<MnemonicDB.Storage.RocksDbBackend.Backend>();
-
         coll.AddSingleton<DatomStoreSettings>(sp =>
             {
                 var fileSystem = sp.GetRequiredService<IFileSystem>();
                 var settingsManager = sp.GetRequiredService<ISettingsManager>();
                 var settings = settingsManager.Get<DataModelSettings>();
+                if (settings.UseInMemoryDataModel)
+                    return DatomStoreSettings.InMemory;
                 return new DatomStoreSettings
                 {
                     Path = settings.MnemonicDBPath.ToPath(fileSystem),
                 };
             }
         );
         
-        coll.AddSingleton<IStoreBackend>(sp =>
-        {
-            var settingsManager = sp.GetRequiredService<ISettingsManager>();
-            var settings = settingsManager.Get<DataModelSettings>();
-            if (settings.UseInMemoryDataModel)
-            {
-                return sp.GetRequiredService<MnemonicDB.Storage.InMemoryBackend.Backend>();
-            }
-            else
-            {
-                var datomStoreSettings = sp.GetRequiredService<DatomStoreSettings>();
-
-                if (!datomStoreSettings.Path.DirectoryExists()) 
-                    datomStoreSettings.Path.CreateDirectory();
```

Comment:
Since this was removed, the App will fail to start from a clean setup, due to:
```
RocksDbSharp.RocksDbException: IO error: No such file or directory: Failed to create a directory: C:/Users/Al12r/AppData/Local/NexusMods.App/DataModel/MnemonicDB.rocksdb: The system cannot find the path specified.
```